code/modules/pai/pai.dm

```diff
@@ -335,6 +335,9 @@
 	master_dna = "Untraceable Signature"
 	// Sets supplemental directive to this
 	laws.supplied[1] = "Do not interfere with the operations of the Syndicate."
+	leash.set_distance(INFINITY) // Freedom!!!
```

Comment:
Delete the component instead.

Reply:
Applied with https://github.com/tgstation/tgstation/pull/77325/commits/33ac110cdd7662596174f575e6a93e967115eb72